packages/cli/src/commands/datasources/pull/index.ts

```diff
@@ -9,13 +9,15 @@ import type { PullDatasourcesOptions } from './constants';
 import { CommandError, handleError, isVitest, konsola, requireAuthentication } from '../../../utils';
 import chalk from 'chalk';
 import { fetchDatasource, fetchDatasources, saveDatasourcesToFiles } from './actions';
+import { join } from 'pathe';
+import { DEFAULT_STORAGE_DIR } from '../../../utils/filesystem';
 
 const program = getProgram();
 
 datasourcesCommand
   .command('pull [datasourceName]')
-  .option('-f, --filename <filename>', 'custom name to be used in file(s) name instead of space id')
-  .option('--sf, --separate-files', 'Argument to create a single file for each datasource')
+  .option('-f, --filename <filename>', 'custom name to be used in file(s) name instead of space id', 'datasources')
```

Comment:
suggestion (non-blocking): We can use a `directories.datasources` constant here.